src/main/java/com/courtcanva/ccfranchise/dtos/FranchiseeGetDto.java

```diff
@@ -0,0 +1,27 @@
+package com.courtcanva.ccfranchise.dtos;
+
+import com.courtcanva.ccfranchise.constants.AUState;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.OffsetDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class FranchiseeGetDto {
+
+    private Long franchiseeId;
+    private String abn;
+    private String businessAddress;
+    private String businessName;
+    private String entityName;
+    private OffsetDateTime createdTime;
+    private OffsetDateTime updatedTime;
```

Comment:
Do we really need to return these two timestamps? What scenarios we're going to use them in?

Reply:
remove redundant timestamps